DataLayer/Repository/IRepository/IRepository.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DataLayer.Repository.IRepository
+{
+    internal interface IRepository
+    {
```

Comment:
why do you push the empty files?

Reply:
Okay, I cloned the repository and added Coutry and City models and pushed it. I have not any branches yet. Did you expect me to write all code in here?